projects/appearances/protos/StonePavement.proto

```diff
@@ -12,34 +13,39 @@ PROTO StonePavement [
 ]
 {
 %{
-  local textureName = fields.type.value:gsub("%s+", "_")
+  io.stderr:write("Deprecated 'StonePavement' PROTO, please use the 'Pavement' PROTO instead and select the right type.\n")
+  if fields.type.value == "recent" then
+    local textureName = "regular"
+  else
+    local textureName = "old_big"
```

Comment:
This is not working because the variable is local to the if and therefore you can't access it later.
This is clearly visible when trying to load it:
```
ERROR: '/home/david/webots-michou/projects/appearances/protos/StonePavement.proto': error: Template engine error: luaL_dostring error : [string "-- template script to parse a procedural node..."]:51: Runtime error in sandboxed code "StonePavement.proto" in line 14:
 attempt to concatenate global 'textureName' (a nil value)

 11:        url [\
 12:          ")
 13:  __liluat_output_function( "\"textures/pavement/" .. textureName .. "_pavement_base_color.jpg\"" )
 14:> __liluat_output_function("\
 15:        ]\
 16:        filtering 5\
 17:      }\.
```